types/pull-stream/pull-stream-tests.ts

```diff
@@ -4,99 +4,133 @@ import pull = require('pull-stream');
  * pull *
  ********/
 
-let source: pull.Source<string> = (end, cb) => undefined;
-let through: pull.Through<string, string> = (source) => source;
-let sink: pull.Sink<string> = (source) => undefined;
+let source: pull.Source<string> = (end, cb) => {};
+let through: pull.Through<string, string> = source => source;
+let sink: pull.Sink<string> = source => {};
+
+function voidFunc(): void {}
+
+type Void = ReturnType<typeof voidFunc>;
 
 // Start with source and end with sink
-let nothing: undefined;
+let nothing: Void;
```

Comment:
Do we really need this? Can't we just use the `void` type?

Reply:
<img width="928" alt="image" src="https://user-images.githubusercontent.com/1164378/90304704-00c8da80-deed-11ea-9681-a1cfedd0c3b1.png">
Because `void` can only be used as a return type.

Reply:
Interesting... In that case forget I said anything 🙂